packages/preload/src/index.ts

```diff
@@ -310,6 +311,35 @@ function initExtensions(): void {
     return getFreePort(port);
   });
 
+  type logFn = (...data: unknown[]) => void;
+
+  contextBridge.exposeInMainWorld('logs', {
```

Comment:
I think logs is too generic there as it's only for providers

`providerLogs` maybe